Successfully updated:
Created Frame node VX8gqCAJU
Created Frame node lkFg_gITU
Created Frame node m3ieov50V
Updated attributes for node VX8gqCAJU
Updated attributes for node lkFg_gITU
Updated attributes for node m3ieov50V
Reordered node lkFg_gITU within parent VX8gqCAJU to index 0

XML Changes:
Index: node.xml
===================================================================
--- node.xml	Before
+++ node.xml	After
@@ -550,40 +550,71 @@
               stackDistribution="center"
               stackAlignment="center"
           >
             <Frame
                 nodeId="eUtmX7NVC"
                 width="100px"
                 height="100px"
                 <!-- background color string or project color style path (if starts with /) -->
                 backgroundColor="rgb(255, 255, 255)"
                 borderRadius="8px"
              />
             <Frame
                 nodeId="KSuoNUJMJ"
                 width="150px"
                 height="80px"
                 <!-- background color string or project color style path (if starts with /) -->
                 backgroundColor="rgb(200, 200, 200)"
                 borderRadius="4px"
              />
           </Stack>
+          <Stack
+              nodeId="VX8gqCAJU"
+              width="100%"
+              height="200px"
+              <!-- background color string or project color style path (if starts with /) -->
+              backgroundColor="rgb(50, 100, 200)"
+              borderRadius="12px"
+              layout="stack"
+              gap="20px"
+              padding="24px"
+              stackDirection="horizontal"
+              stackDistribution="center"
+              stackAlignment="center"
+          >
+            <Frame
+                nodeId="lkFg_gITU"
+                width="100px"
+                height="100px"
+                <!-- background color string or project color style path (if starts with /) -->
+                backgroundColor="rgb(255, 255, 255)"
+                borderRadius="8px"
+             />
+            <Frame
+                nodeId="m3ieov50V"
+                width="150px"
+                height="80px"
+                <!-- background color string or project color style path (if starts with /) -->
+                backgroundColor="rgb(200, 200, 200)"
+                borderRadius="4px"
+             />
+          </Stack>
         </Header>
         <Actions
             nodeId="XQhemvRcY"
             width="fit-content"
             height="fit-content"
             overflow="visible"
             layout="stack"
             gap="12px"
             stackDirection="horizontal"
             stackDistribution="center"
             stackAlignment="center"
         >
           <Button
               nodeId="WrBnmQyyQ"
               width="fit-content"
               height="fit-content"
               <!-- the component id this instance uses -->
               componentId="zQ4rhDKqX"
               variant="anmz3_Hx4"
               XNZ6TQLVf="Read the Docs"
@@ -3898,36 +3929,42 @@
       width="fit-content"
       height="fit-content"
       top="0px"
       left="0px"
       font="Inter"
       textTruncation="2"
   >
     Truncated Text that is long enough to be truncated
   </TruncatedTextThatIsLongEnoughToBeTruncated>
 </Frame>
 <Frame
     nodeId="yKtdVfGM5"
     position="absolute"
     width="100px"
     height="100px"
     top="0px"
     left="0px"
     <!-- background color string or project color style path (if starts with /) -->
     backgroundColor="rgb(200, 200, 200)"
  />
-<Stack
-    nodeId="xewAIW2EI"
+<Frame
+    nodeId="uDmkNrg_V"
     position="absolute"
-    width="400px"
-    height="300px"
+    width="200px"
+    height="200px"
     top="0px"
     left="0px"
     <!-- background color string or project color style path (if starts with /) -->
-    backgroundColor="rgb(240, 240, 240)"
-    layout="stack"
-    gap="16px"
-    padding="20px"
-    stackDirection="vertical"
-    stackDistribution="center"
-    stackAlignment="center"
- />
+    backgroundColor="rgb(200, 200, 200)"
+>
+  <TruncatedTextThatIsLongEnoughToBeTruncated
+      nodeId="I3ywfKHsz"
+      position="absolute"
+      width="fit-content"
+      height="fit-content"
+      top="0px"
+      left="0px"
+      font="Inter"
+  >
+    Truncated Text that is long enough to be truncated
+  </TruncatedTextThatIsLongEnoughToBeTruncated>
+</Frame>


Note: Set zoomIntoView=false if you want to use Framer app while MCP is working.